doc/explanation/index.md

```diff
@@ -3,20 +3,53 @@
 
 The explanatory guides in this section introduce you to the concepts used in LXD and help you understand how things fit together.
 
+## Important concepts
+
+Before you start working with LXD, you need to be familiar with some important concepts about LXD and the instance types it provides.
+
+```{toctree}
+:titlesonly:
+
+/explanation/lxd_lxc
+/explanation/instances
+```
+
+## Entities in LXD
+
+When working with LXD, you should have a basic understanding of the different entities that are used in LXD.
+See the {ref}`howtos` for instructions on how to work with these entities, and the following guides to understand the concepts behind them.
+
+```{toctree}
+:titlesonly:
+
+/image-handling
+About storage </explanation/storage>
+/explanation/networks
+/database
+```
+
+## Access management
+
+LXD does not have the concept of *users*.
```

Comment:
This section is quite right any more. We still don't have users, but we do have Identities (which for many cases are equivalent to Users) and these that don't have to use TLS authentication, they can use OIDC authentication.